src/styles/03-organisms/iconlink-list/_iconlink-list.scss

```diff
@@ -0,0 +1,12 @@
+.iconlink-list {
+  h2 {
```

Comment:
Instead of attaching styles to the elements, please attach them to their blocknames. E.g.,

```
.iconlist-list {
  &__heading {
    @include space-stack-none;
  }

  &__list {
    color: $gray-medium;
  }
}
```

Etc.